### PR TITLE
RELATED: BB-1703 measure value filter docs

### PR DESCRIPTION
**Pull request by strouhaljan**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/161/ : SUCCESS in 1m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/41/ : SUCCESS in 6s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/164/ : SUCCESS in 2m 10s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/44/ : SUCCESS in 7s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/165/ : SUCCESS in 47s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/45/ : SUCCESS in 7s

File: docs/measure_value_filter_component.md [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+  render() {
+    const { filter } = this.state;
+    return (
+      <div>
+        <MeasureValueFilterDropdown
+          onApply={this.onApply}
+          measureTitle={totalSales.measure.title}
+          measureIdentifier={totalSales.measure.localIdentifier}
+          filter={filter || null}
```
Comment (yanlipnican) [inline docs/measure_value_filter_component.md]: I feel like this is unnecessary

File: docs/measure_value_filter_component.md [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+  .title("$ Total Sales");
+
+const locationResort = Model.attribute(
+  "locationResortIdentifier"
+).localIdentifier("locationResort");
+
+export default class SalesByResort extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { filter: null };
```
Comment (yanlipnican) [inline docs/measure_value_filter_component.md]: you can skip constructor and do just ```state = { filter: null }```

**Comment by yanlipnican** [comment]
LGTM

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/166/ : SUCCESS in 1m 40s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-oss-check/46/ : SUCCESS in 8s

**Comment by gdgate** [comment]
Build succeeded (gate pipeline).

- https://checklist.intgdc.com/job/client-libs/job/gooddata-ui-documentation-validation/167/ : SUCCESS in 1m 19s
- https://checklist.intgdc.com/job/compliance/job/gooddata-ui-merge-compliance-check/45/ : SUCCESS in 22s

**Pull request merged**